grunttasks/changelog.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (grunt) {
     options: {
       // the empty object is needed or else conventionalChangelog blows up.
       changelogOpts: {
+        preset: 'angular'
```

Comment:
I think we need to kill the comment on line 8 now that we aren't passing an empty object.